gulpfile.ts

```diff
@@ -229,10 +258,11 @@ export class Gulpfile {
     package() {
         return [
             "clean",
-            ["browserCopySources", "browserCopyMainBrowserFile", "browserCopyPlatformTools"],
-            ["packageCompile", "browserCompile"],
+            ["browserCopySources", "browserCopyPlatformTools", "browserCopyMainBrowserFile"],
```

Comment:
you probably aware that all 3 execute in parallel and order of these does not matter?

Reply:
Yes I am aware of this, this change comes because I had a version that wouldn't need the `browserCopyMainBrowserFile` so I removed it and later added it back in but at a different place in the list. I will revert that change